com.unity.ml-agents.extensions/Editor/Input/InputActuatorComponentEditor.cs

```diff
@@ -0,0 +1,26 @@
+#if MLA_INPUT_SYSTEM && UNITY_2019_4_OR_NEWER
+using Unity.MLAgents.Extensions.Input;
+using UnityEditor;
+
+namespace Unity.MLAgents.Extensions.Editor.Input
+{
+    [CustomEditor(typeof(InputActuatorComponent))]
+    internal class InputActuatorComponentEditor : UnityEditor.Editor
+    {
+        const string k_ActionSpecName = "m_ActionSpec";
+
+        public override void OnInspectorGUI()
+        {
+            var so = serializedObject;
+            so.Update();
+            InputActuatorComponent o = so.targetObject as InputActuatorComponent;
+            _ = o.ActionSpec;
+            EditorGUI.indentLevel++;
+            EditorGUI.BeginDisabledGroup(true);
```

Comment:
Just for clarification, this always appears disabled for editing, and just shows the ActionSpec for the component?

Reply:
yup